fasstq_filtrator/fastq_filtrator.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+
+def filtrator (args):
+     with open(f"{args['output_file_prefix']}_passed.fastq", "w") as passed_read:
+        if args['save_filtered']:  
+            with open(f"{args['output_file_prefix']}_failed.fastq", "w") as failed_read:
+                with open(args['input_fastq'], "r") as fastq_file:
+		#reading fastq string-by-string 
+                    while True:
+                        seq_name = fastq_file.readline()
+                        seq = fastq_file.readline()
+                        seq_length = len(seq[:-1])
+                        string_3 = fastq_file.readline()
+                        quality = fastq_file.readline()
+                        if seq_name == "":
+                            break
+		#gc fraction calculation 
+                        gc_amount = seq.count("C") + seq.count("G")
+                        gc_fraction = gc_amount / seq_length * 100
+                        if not args['gc_bounds'][0] <= gc_fraction <= args['gc_bounds'][1]:
+                            failed_read.write(seq_name)
+                            failed_read.write(seq)
+                            failed_read.write(string_3)
+                            failed_read.write(quality)
+                            continue 
+                  #length bounds range      
+                        if not args['length_bounds'][0] <= seq_length <= args['length_bounds'][1]:
+                            failed_read.write(seq_name)
+                            failed_read.write(seq)
+                            failed_read.write(string_3)
+                            failed_read.write(quality)
+                            continue 
+		#quality counter check 
+
+                        quality_counter = 0
+                        for item in quality[:-1]:
+                            quality_counter+=int(ord(item))
```

Comment:
`ord` уже возвращает целое число, так что преобразование излише. Также ты забыла перевести качество в шкалу phred33
```suggestion
                            quality_counter += ord(item) - 33
```